Sources/PassKit/PassKit.swift

```diff
@@ -440,7 +448,7 @@ public class PassKitCustom<P, D, R: PassKitRegistration, E: PassKitErrorLog> whe
             
             let data = try Data(contentsOf: file)
             let hash = Insecure.SHA1.hash(data: data)
-            manifest[relativePath] = hash.description
+            manifest[relativePath] = hash.map { String(format: "%02hhx", $0) }.joined()
```

Comment:
Ack, no no no, don't do this, printf formats are not a good idea in Swift 😅. Either use Vapor's hex utility (`Array(hash).hex` should do the trick), or if you want a one-liner for it in pure stdlib, try `hash.map { "0\(String($0, radix: 16))".suffix(2) }.joined()` (Vapor's version is more efficient, though).